pyramid/file.go

```diff
@@ -0,0 +1,37 @@
+package pyramid
```

Comment:
![I like the pyramid at Uxmal](https://user-images.githubusercontent.com/7802932/100247526-1796b880-2f43-11eb-9bff-8672c68b8ebb.png)
:smile: